README.md

```diff
@@ -52,14 +52,23 @@ Want to chat with other members of the CoreFX community?
 
 [.NET Foundation forums]: http://forums.dotnetfoundation.org/
 
+### Reporting security issues and bugs
+
+Security issues and bugs should be reported privately, via email, to the
```

Comment:
Nit: could you make it "security issues and security bugs"?  Otherwise it could interpreted as "(security issues) and bugs" instead of the intended "security (issues and bugs)".